components/landing/HeroBanner.vue

```diff
@@ -16,6 +31,20 @@
           {{ $t('search.landingTitle3') }}
         </span>
       </h1>
+      <template v-else>
+        <h1
+          class="title is-size-1 is-size-2-mobile has-text-weight-bold has-text-centered flex flex-wrap justify-center items-center mb-0">
+          {{ $t('search.landingTitle1') }}
+          <span
+            class="subtitle is-size-1 is-size-2-mobile has-text-weight-bold has-text-centered capitalize ml-4">
+            {{ $t('search.landingTitle2') }}
+          </span>
+        </h1>
+        <h1
+          class="is-size-1 is-size-2-mobile has-text-weight-bold has-text-centered">
+          {{ $t('search.landingTitle3') }}
+        </h1>
```

Comment:
multiple `h1` 

**suggestion:**: wrap text in `span` instead

Reply:
updated